src/main/java/com/task10/crudapi_login/mapper/ClientMapper.java

```diff
@@ -1,18 +1,23 @@
 package com.task10.crudapi_login.mapper;
 
 import com.task10.crudapi_login.entity.Client;
+import org.apache.ibatis.annotations.Insert;
 import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Options;
 import org.apache.ibatis.annotations.Select;
 
 import java.util.List;
 import java.util.Optional;
 
-
 @Mapper
 public interface ClientMapper {
     @Select("SELECT * FROM clients ")
     List<Client> findAll();
 
     @Select("SELECT * FROM clients WHERE id = #{id}")
     Optional<Client> findById(int id);
+
+    @Insert("INSERT INTO clients(name, age, phoneNumber) VALUES (#{name}, #{age}, #{phoneNumber})")
+    @Options(useGeneratedKeys = true, keyProperty = "id")
+    void insertClient(Client client);
```

Comment:
細かいですが、ClientMapperの関数なので、`insert`という関数名の方が意味の重複がなくて良さそうです！
```suggestion
    void insert(Client client);
```

Reply:
修正いたしました！
fef6a6aed9a988a40cf628d19370dfb070faa790